pset-iterators.js

```diff
@@ -160,9 +182,11 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
-
+const firstRepublican = presidents.find((president) => {
+  return president.party === "Republican"
+})
 // Add your code above this line
-
+console.log(firstRepublican)
```

Comment:
@jl1982 Great work here using the find method to return the first republican party. The find method is very valuable when you only want to return the first occurrence of a value in an array.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)